docs/source/advanced_topics/11_memory_representation_function.rst

```diff
@@ -0,0 +1,50 @@
+Conversions Between GPU Memory Formats
+---------------------------------------------
+
+When working with images, there are many ways to represent them as arrays of pixels. Working with different models you may encounter representation of an image using OpenCV Mat class, PyTorch tensor or CuPy array.
+
+The Savant framework aims to maximize GPU utilization without unnecessary data copying and conversion. To achieve this, Savant provides functions for converting between different image representations. Data exchange is performed with zero-copying between different views, except for some cases of conversion to GpuMat OpenCV.
```

Comment:
I would say it minimized transactional costs rather than maximized GPU utilization. We normally do want GPU to be utilized less to compute more data. So we do not have a purpose to maximize GPU utilization. We maximize processing efficiency.

Reply:
From my point of view, maximizing GPU utilization means maximizing the number of computations that can be performed on the GPU. These functions allow the user not to transfer data to the CPU and perform calculations and transformations that he needs to do with PyTorch or CuPy on the GPU. If you have any suggestions on how to write it better I'd be only too happy.

Reply:
@dorgun ok,
"The Savant framework aims to use GPU efficently without excessive data transfers..."

Reply:
Fixed